### PR TITLE
Register krish.is-a.dev

### PR DESCRIPTION
**Pull request by krish858**
Register krish.is-a.dev with CNAME record pointing to krish858.github.io.

### CODE REVIEW

File: domains/krish.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "krish858",
+           "email": "unknowmortal57@gmail.com",
+           "discord": "789102469345968131"
+        },
+    
+        "record": {
+            "CNAME": "krish858.github.io"
```
Comment (CuteDog5695) [inline domains/krish.json:9]: 404 page, please give a preview of your website

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/krish.json (hunk 1 above)
Comment (krish858) [reply domains/krish.json:9]: Preview: krish858.github.io/portfolio

Comment (krish858) [reply domains/krish.json:9]: > 404 page, please give a preview of your website

preview:%20krish858.github.io/portfolio

Comment (21Z) [reply domains/krish.json:9]: Complete your website

Comment (CuteDog5695) [reply domains/krish.json:9]: What 21Z said.

**Comment by krish858** [comment]
> Your CNAME contains a `/`. That isn't allowed.

Updated

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**